scripts/contractInteraction/mainnet_contracts.json

```diff
@@ -27,5 +27,21 @@
 
     "CSOV1": "0x0106F2fFBF6A4f5DEcE323d20E16E2037E732790",
     "CSOV2": "0x7f7Dcf9DF951C4A332740e9a125720DA242A34ff",
-    "governorVault": "0xC7A1637b37190a456b017897207bceb2A29f19b9"
+    "governorVault": "0xC7A1637b37190a456b017897207bceb2A29f19b9",
+
+    "SOV": "0xEFc78fc7d48b64958315949279Ba181c2114ABBd",
```

Comment:
Should we make it checksum valid?

https://explorer.rsk.co/check-address/0xEFc78fc7d48b64958315949279Ba181c2114ABBd
vs
https://explorer.rsk.co/check-address/0xefc78fc7d48b64958315949279ba181c2114abbd